### DISCUSSION TITLE
Plugin display cropped

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#785] [Help]
Hi,

So I already noticed that for some plugins : a big part of the plugin is hidden under the top of the agridder frame : so only the bottom of the plugin  is displayed and the bottom of the AG frame is black. The top of the pugin is missing.

Looks like
![Screenshot 2021-11-04 at 18 07 38](https://user-images.githubusercontent.com/56123216/140386673-4dc8b488-7288-4ecd-9f9c-06e7fdc7e81e.png)
 wrong top offset for grabbing the plugin window maybe ?

Here is an example with Brighton compressor (N4 Library with free trial)

**Comment by Henri75** [comment]
By the way there is still the offset issue : Remote mouse position is not the same than the one on the local display (something like 10-20px vertical offset). So to click on something you need to click 20px above it !!
Doesnt seems to happen on mac server, only on windows server (4 different ones running latest windows 10 update)

**Comment by Henri75** [comment]
The cropped plugin window in local mode (mac) happens with a lot of plugins, and 100% of Acustica Nebula N4 plugins or third party libraries (like brighton).
Doesnt happen when you connect to a remote mac, only on local server !

**Comment by CaBuffalo** [comment]
Can you send screenshot or log files? That's more helpful, Thanks.

On Mon, Nov 22, 2021 at 8:37 AM Henri75 ***@***.***> wrote:

> The cropped plugin window in local mode (mac) happens with a lot of
> plugins, and 100% of Acustica Nebula N4 plugins or third party libraries
> (like brighton).
> Doesnt happen when you connect to a remote mac, only on local server !
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/785#discussioncomment-1682724>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CODCFRVMCZB2GVTWLUNJWVLANCNFSM5HL6PAKA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Henri75** [comment]
There is one at the beginning of the discussion.
Here is an other example : on the left the plugin inserted in DAW, on the right the plugin inside AG !
![Screenshot 2021-11-22 at 19 28 26](https://user-images.githubusercontent.com/56123216/142916103-5c96f575-99c5-4420-95a7-4a8667d68fdd.png)

**Reply by giacometti777** [reply]
I just want to note I have the exact same cropping issue with all Nebula libs but only in remote connections - even when used as a VST "setup". It happens whether the server is on an Intel Mac or a Windows PC (I happen to have both in my workflow). It seems like just drawing N4 plugins down 800px or so would solve the issue functionally in most cases. It would be so appreciated if a simple fix like this went into the next beta!

**Comment by oldtos** [comment]
I think in the upper right corner near the top on the plugin (daw pc audio gridder plugin)  there is a + and - and another symbol... if I recall that is what fixed the issue for me... poke at them.

**Reply by giacometti777** [reply]
Unfortunately because the majority of the interface is hidden above the top of the Audiogridder instance window no amount of toggling those buttons helps with Nebula - though I have had success in getting access to hidden parts of other plugins! This seems to be a Nebula specific issue.

**Reply by oldtos** [reply]
maybe generic editor .. its not pretty ,,, but ... (mixer looking bit next to sever name)

**Reply by giacometti777** [reply]
It definitely works for some libraries but others don't have every parameter controllable on the generic editor. Thank you for the tips, however!
